conda_env/cli/main.py

```diff
@@ -35,6 +35,14 @@
 from . import main_update
 
 
+# TODO: This belongs in a helper library somewhere
```

Comment:
No -- you want to circumvent getting to the point of `args_func` by injecting the `--help` in to the actual args so it's there before its parsed.